CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+# dbt_shopify v0.15.0
+
+[PR #94](https://github.com/fivetran/dbt_shopify/pull/94) includes the following updates:
+## Breaking Changes
+- Updated columns with the connector changes released on January 6, 2025. See the [release notes](https://fivetran.com/docs/connectors/applications/shopify/changelog#january2025) for more details. 
+
+- In the `shopify__inventory_levels` model, replaced the `cost` column with:
+  - `unit_cost_amount`
+  - `unit_cost_currency_code`
+
+- Added the following columns to models:
+  - `shopify__inventory_levels`:
+    - `duplicate_sku_count`
+    - `harmonized_system_code`
+    - `inventory_history_url`
+    - `legacy_resource_id`
+    - `measurement_id`
+    - `measurement_weight_value`
+    - `measurement_weight_unit`
+    - `is_tracked_editable_locked`
+    - `tracked_editable_reason`
+  - `shopify__order_lines`:
```

Comment:
looks like the columns that were added to `shopify__order_lines` were also added into `shopify__inventory_levels`, should we make this tweak:

```suggestion
  - `shopify__inventory_levels` and `shopify__order_lines`:
```